dcos/dcos-ui
# DCOS_OSS-4957: clarify restart configuration

Clarify layout of restart configuration in jobs form.

Closes DCOS_OSS-4957.

## Testing

See [designs](https://mesosphere.invisionapp.com/share/Y6Q8EFKBA3G#/screens/345675728)
- Open Jobs form
- Check that tab names match designs (it is known that the tab labels will wrap onto a new line on smaller screen sizes. Design is aware)
- Go to Run Configuration tab
- Under Restart Policy heading, there should be no "Restart Job" label above radio buttons, and radio button labels should be "Never/On Failure" instead of "Yes/No". "Retry Time" label has changed to "Keep Trying Time"
- If "On Failure" is not selected for Restart Policy, Keep Trying Time input should be disabled (it's ok if the activeDeadlineSeconds is not removed from the JSON)

## Trade-offs

N/A

## Dependencies

N/A

## Screenshots

Before
<img width="332" alt="Screen Shot 2019-04-03 at 8 47 46 AM" src="https://user-images.githubusercontent.com/19582796/55494082-f905e000-55ee-11e9-9a1d-3ab9da7a5ce0.png">

After
<img width="659" alt="Screen Shot 2019-04-04 at 11 41 27 AM" src="https://user-images.githubusercontent.com/19582796/55580109-a7cd1d80-56ce-11e9-9a75-ebfe80ae6eca.png">
